mk/python.mk

```diff
@@ -0,0 +1,12 @@
+PYTHON = python3
+PYTHON := $(shell which $(PYTHON))
+ifndef PYTHON
+$(error "python3 is required.")
+endif
+
+# check "iota" module in Python installation
+PY_CHECK_MOD_IOTA := $(shell $(PYTHON) -c "import iota" 2>/dev/null && \
+                       echo 1 || echo 0)
+ifneq ("$(PY_CHECK_MOD_IOTA)","1")
+    $(error "skip $@ because PyIOTA is not installed.")
```

Comment:
It is `PyOTA` or `iota.lib.py`.

Reply:
The error message is wrong because it does not "skip". Instead, we would take actions if both Python3 and PyOTA are absent. You shall dump informative diagnosis messages.